### DISCUSSION TITLE
Stochasticity in Practical 3-4

### DISCUSSION THREAD

**Discussion by mhuzzell** [IBAHCM/RPiR#544] [Q&A for practicals] [answered]
I just wanted to check my understanding of the instructions. It looks like we are meant to write a stochastic function for new transmissions, but assume a constant (deterministic) rate of recovery? This would let the total population continue to add up to the right numbers, which seems sensible, but I can see from looking at other people's code that some have taken a different approach, so just wanted to see if I've got that wrong.

**Comment by richardreeve** [comment, marked as answer]
The rate of recovery is a constant (at least per infected individual for a given timestep), but the outcome is still stochastic because that rate is used as a probability of recovering ("success") in a binomially distributed random number draw. Does that make sense?
